From f6e88deab23181a6c48828250b306627a80404b7 Mon Sep 17 00:00:00 2001
From: Marcel Metz <mmetz@adrian-broher.net>
Date: Tue, 6 Sep 2016 16:59:04 +0200
Subject: [PATCH] Disable MSVC specific compiler warnings

* Disable C4101 'identifier' : unreferenced local variable for libpython.
* Disable C4018 'expression' : signed/unsigned mismatch for libpython.
* Disable C4244 'conversion' conversion from 'type1' to 'type2', possible
  loss of data for libpython.
* Disable C4996 The compiler encountered a deprecated declaration. for
  libpython.
---
 cmake/libpython/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cmake/libpython/CMakeLists.txt b/cmake/libpython/CMakeLists.txt
index 7a089a8..a33084f 100644
--- a/cmake/libpython/CMakeLists.txt
+++ b/cmake/libpython/CMakeLists.txt
@@ -178,6 +178,9 @@ elseif(WIN32)
             Py_ENABLE_SHARED
             MS_DLL_ID="${ms_dll_id}"
         )
+    if(MSVC)
+        add_compile_options(/wd4101 /wd4018 /wd4244 /wd4996)
+    endif()
 endif()
 
 set(THREAD_SOURCES )
-- 
2.14.3

